index.html

```diff
@@ -19,8 +45,8 @@ <h2 class="title-text">All Awesome Books</h2>
         <tbody>
         </tbody>
       </table>
-    </div>
-    <div id="add-book-form">
+    </section>
+    <section id="add-book-form" class="content-section hidden">
```

Comment:
- [ ] For this section, I think you can still improve the design to match what we have in the wireframe. The structure is not as expected. I think you can use `css flexbox` to achieve this structure with a `flex-direction` set to `column`.
Check it out :pray:: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/235948248-a06dc6ee-380d-463d-af66-661fe28760c5.png)|![image](https://user-images.githubusercontent.com/99715640/235948391-91332ba1-80d1-456a-a491-a1faa5bb56c0.png)|
  - [ ] Kindly remember that the time & date feature should be seen at the top here too
  - [ ] As in the wireframe, it will be okay to have the content centralized so that the spaces on both sides are even
  - [ ] All `input` elements are best positioned on separate rows to have a classic view.